gc/base/standard/Scavenger.cpp

```diff
@@ -3011,11 +3011,12 @@ MM_Scavenger::backOutFixSlot(GC_SlotObject *slotObject)
 		MM_ForwardedHeader forwardHeader(objectPtr, OMR_OBJECT_METADATA_SLOT_OFFSET);
 		Assert_MM_false(forwardHeader.isForwardedPointer());
 		if (forwardHeader.isReverseForwardedPointer()) {
-			omrobjectptr_t fwdObjectPtr = forwardHeader.getReverseForwardedPointer();
+			slotObject->writeReferenceToSlot(forwardHeader.getReverseForwardedPointer());
 #if defined(OMR_SCAVENGER_TRACE_BACKOUT)
+			OMRPORT_ACCESS_FROM_OMRVM(_omrVM);
+			omrtty_printf("{SCAV: Back out object slot %p[%p->%p]}\n", slotPtr, objectPtr, *slotPtr);
 			Assert_MM_true(isObjectInEvacuateMemory(fwdObjectPtr));
```

Comment:
I believe there is a compile failure here if you compile with the debug code. fwdObjectPtr is no longer declared.